lib/functions.js

```diff
@@ -0,0 +1,28 @@
+// Focused on HU03 and HU02
+function recommendSong(playlist, song) {
+	const genresWeight = 75;
```

Comment:
¿Por qué asignas estos valores a los pesos?

Reply:
Probando varios valores estos han sido los que mejor se ajustaban para pasar el `test`. Ya que primero se hizo la prueba y luego se arregló el código para que se ajuste y no al revés.

Además tiene sentido, ya que con que sea una colaboración se considera que es una buena recomendación y depende del género puede ser buena, no serlo o incluso se puede llegar a discernir si una recomendación es mejor o no, ya que el número que devuelve indica eso.

Reply:
Para empezar, eso es lo mismo que 3 y 2 o 1.5 y 1. Si usas valores arbitrarios, tienes que justificarlos bien, sobre todo mirando a la historia de usuario.

Reply:
Si en la historia de usuario no  tengo nada relacionado con los valores que tomen, ¿cómo puedo hacerlo?

Reply:
Pero entonces no sé cómo voy a recomendar las canciones... Debería añadir o modificar las HU ya existentes para que incluya algo así, porque sino, no se me ocurre cómo puedo discernir si es una buena o mala recomendación.

Como el proyecto va desarrollándose dinámicamente supongo que se podría añadir una historia de usuario para explicar que el peso del género y el de las colaboraciones debe ser distinto para que se pueda obtener una recomendación mejor.

Reply:
> Pero entonces no sé cómo voy a recomendar las canciones... Debería añadir o modificar las HU ya existentes para que incluya algo así, porque sino, no se me ocurre cómo puedo discernir si es una buena o mala recomendación.
>


Álvaro, es que eso es lo que os llevo pidiendo desde el objetivo 0. Que entendáis el problema, y tengáis una ligera idea de como resolverlo. No puedes modificar las HU para que pidan lo que ya estás haciendo. Tienes que resolver el problema que se muestra en la HU, de eso va toda la metodología.
 
> Como el proyecto va desarrollándose dinámicamente supongo que se podría añadir una historia de usuario para explicar que el peso del género y el de las colaboraciones debe ser distinto para que se pueda obtener una recomendación mejor.

Son detalles de implementación. Si quieres usarlo, tendrás que plantear por qué necesitas tener en cuenta esas dos cosas y para qué historia de usuario.